content/en/APIUseCases/create_asset.md

```diff
@@ -0,0 +1,229 @@
+---
+title: "Use Our API to Create an Asset"
+linkTitle: "Create an Asset"
+weight: 110
+description: >
+  Use this sequence of REST calls to create an asset.
+---
+
+{{% pageinfo %}}
+Use this document with our [Cobalt API documentation](https://docs.cobalt.io) to
+manage assets and pentests.
+{{% /pageinfo %}}
+
+<!-- Future task: set up variables for `YOUR-PERSONAL-API-TOKEN` and
+`YOUR-V2-ORGANIZATION-TOKEN`. May support automated populating of REST calls. -->
+
+This page describes how you can use our API To create an [asset](../../getting-started/glossary/#asset). 
+
+## Create an API Token in the Cobalt UI
```

Comment:
Option:

## Create an API Token in the Cobalt App

Reply:
I'm going to say "no" to this, as I've actually found an (undocumented) REST call to create an API Token.

Reply:
No problem.